packages/better-php-doc-parser/src/ValueObject/PhpDocNode/AbstractTagValueNode.php

```diff
@@ -189,8 +189,10 @@ private function shouldPrintEmptyBrackets(): bool
         )) {
             return true;
         }
-
-        return $this->tagValueNodeConfiguration->hasOpeningBracket() && $this->tagValueNodeConfiguration->hasClosingBracket();
+        if (! $this->tagValueNodeConfiguration->hasOpeningBracket()) {
+            return false;
+        }
+        return (bool) $this->tagValueNodeConfiguration->hasClosingBracket();
```

Comment:
`hasClosingBracket()` returns bool, so type-casting is not needed here

Reply:
implemented.